TACC/Core-Portal
# Bug/WP-24: Disabling Google Drive Integration

## Overview
Updated Task: Disabling Google Drive Integration as it is blocked by [WP-342](https://tacc-main.atlassian.net/browse/WP-342)
Original Task: Selecting a file in Google Drive to copy, allows you to try to copy to google drive which fails.  We need to disable copying to google drive.

## Related

* [WP-24](https://tacc-main.atlassian.net/browse/WP-24)

## Changes
- Disabled Google Drive setup integration on the front end temporarily 
- Disabled Google Drive in the Data Files dropdown menu
- Added data length check for if there are no other 3rd party app integration settings to display 
"No integrations available."
- Adjusted tests to conform the above two being disabled
- Adjusted fixture and DataListingCells call for Google Drive to be disabled


## Testing
For general testing
1. Navigate to Data Files and make sure Google Drive is not a listed system.
2. Try to copy a file and make sure that Google Drive is not a shared system in the dropdown.

For Unit Tests
1. Navigate to `Core-Portal/client/src/components/ManageAccount`
3. Run `npx vitest run` to run local component tests relating to Google Drive
4. Run `npm run test` to run all frontend tests

## UI
Before
![image](https://github.com/user-attachments/assets/f1b6c639-4408-4a7e-8b71-396f5a0bbe0e)
![image](https://github.com/user-attachments/assets/088d7c21-c689-4aa0-aec1-bf16fe3dae2b)

After
![image](https://github.com/user-attachments/assets/89708367-7b66-4e8e-ad79-edbc9b6b4807)
![image](https://github.com/user-attachments/assets/62ee8105-bec9-4890-891e-cf65a5c372b2)
